BasePolicies/README.md

```diff
@@ -0,0 +1,42 @@
+## Establishing a new Working Group
+
+A Working Group is established by first defining a charter that can be
+ratified by the TC. A charter is a *statement of purpose*, a
+*list of responsibilities* and a *list of initial membership*.
```

Comment:
Nit: The Charter template does not seem to include a statement of purpose or a list of responsibilities. It would be ideal if the charter template lined up very clearly with the elements of a charter described here. (But I wouldn't hold this up just for that. It can be done as a subsequent improvement.)